backend/internal/userschema/service/userschemaservice.go

```diff
@@ -42,6 +43,9 @@ type UserSchemaServiceInterface interface {
 	UpdateUserSchema(schemaID string, request model.UpdateUserSchemaRequest) (
 		*model.UserSchema, *serviceerror.ServiceError)
 	DeleteUserSchema(schemaID string) *serviceerror.ServiceError
+	ValidateUser(userType string, userAttributes json.RawMessage) *serviceerror.ServiceError
+	ValidateUserUniqueness(userType string, userAttributes json.RawMessage,
```

Comment:
```suggestion
	IsUniqueUser(userType string, userAttributes json.RawMessage,
```
Suggestion

Reply:
Would like to keep the validate part, since this not a simple unique check..